### PR TITLE
Updated instructions for CUDA 10 and new bluehive modules

### PR DESCRIPTION
**Pull request by whitead**
Please checkout these new simpler installation instructions in sphinx-docs/source/compilations.rst

### CODE REVIEW

File: sphinx-docs/source/compilation.rst [hunk 1]
```diff
@@ -9,25 +9,20 @@ Load the modules necessary:
 
 .. code:: bash
 
-    module load git anaconda3/2018.12b cmake sqlite cudnn/9.0-7
+    module load tensorflow/1.15.0/b1 git
```
Comment (mehradans92) [inline sphinx-docs/source/compilation.rst]: add cmake

**Comment by whitead** [comment]
Notes to me: add flag `-DCMAKE_INSTALL_PREFIX=`python3 -c "import site; print(site.getsitepackages()[0])"`` to hoomd-blue instructions and/or just link to hoomd-blue compilation instructions.

**Comment by whitead** [comment]
Current build strategy (also in compilation instructions)

```bash

module load tensorflow/1.15.0/b1 git cmake
conda create -n hoomd-tf python=3.7
source activate hoomd-tf
python -m pip install tensorflow-gpu==1.15.0
conda install -c conda-forge hoomd==2.5.2
git clone https://github.com/ur-whitelab/hoomd-tf
cd hoomd-tf && mkdir build && cd build
CXX=g++ CC=gcc cmake ..
make install
cd .. && python htf/test-py/test_sanity.py
```

@mehradans92 @RainierBarrett please re-review

**Comment by hgandhi2411** [comment]
@whitead You might want to add the extra cmake flags in the install instructions.

**Comment by RainierBarrett** [comment]
Besides two PermissionErrors, which will be addressed in #131, I got all the tests to run and pass!

**Comment by mehradans92** [comment]
All tests passed. Maybe tensorflow-gpu could be included in module tensorflow/1.15.0/b1 so we don't have to install it.

**Pull request merged**